Tests/VideoTests.swift

```diff
@@ -1,100 +1,211 @@
-import XCTest
+import Nimble
 @testable import ParselyAnalytics
+import XCTest
 
 class VideoTests: ParselyTestCase {
+
     let testVideoId: String = "videoId"
     let testUrl: String = "testurl"
-    var videoManager: VideoManager?
-
-    override func setUp() {
-        super.setUp()
-        videoManager = VideoManager(trackerInstance: parselyTestTracker)
+    var testVideoKey: String {
+        return "\(testUrl)::\(testVideoId)"
     }
 
     func testTrackPlay() {
-        XCTAssertEqual(videoManager!.trackedVideos.count, 0,
-                       "videoManager.accumulators should be empty before calling trackPlay")
-        videoManager!.trackPlay(url: testUrl, urlref: testUrl, vId: testVideoId, duration: TimeInterval(10),
-                                metadata: nil, extra_data: nil, idsite: Parsely.testAPIKey)
-        XCTAssertEqual(videoManager!.trackedVideos.count, 1,
-                       "A call to trackPlay should populate videoManager.accumulators with one object")
+        let videoManager = VideoManager(trackerInstance: parselyTestTracker)
+
+        expect(videoManager.trackedVideos).to(beEmpty())
+
+        videoManager.trackPlay(
+            url: testUrl,
+            urlref: testUrl,
+            vId: testVideoId,
+            duration: TimeInterval(10),
+            metadata: nil,
+            extra_data: nil,
+            idsite: Parsely.testAPIKey
+        )
+
+        expect(videoManager.trackedVideos).to(haveCount(1))
     }
 
     func testTrackPause() {
-        videoManager!.trackPlay(url: testUrl, urlref: testUrl, vId: testVideoId, duration: TimeInterval(10),
-                                metadata: nil, extra_data: nil, idsite: Parsely.testAPIKey)
-        videoManager!.trackPause()
-        XCTAssertEqual(videoManager!.trackedVideos.count, 1,
-                       "A call to trackPause should not remove an accumulator from videoManager.accumulators")
+        let videoManager = VideoManager(trackerInstance: parselyTestTracker)
+
+    videoManager.trackPlay(
```

Comment:
This line doesn't have the right indentation. Move one more level to the right?

Reply:
🤦‍♂️ there's something clashing between my VS Code and `.vimrc` and it's breaking formatting 😞